app/routing/htmlRoutes.js

```diff
@@ -0,0 +1,18 @@
+var bodyParser = require("body-parser");
+var path = require("path");
+
+module.exports = function(app) {
+	// Basic route that sends the user first to the AJAX Page
+	app.get("/", function(req, res) {
+	  res.sendFile(path.join(__dirname, "../public/home.html"));
+	});
+
+	app.get("/survey", function(req, res) {
+	  res.sendFile(path.join(__dirname, "../public/survey.html"));
+	});
+
+	// app.get("/all", function(req, res) {
+	//   res.sendFile(path.join(__dirname, "all.html"));
+	// });
```

Comment:
this doesn't seem necessary, you wanna try to avoid pushing commented code that doesn't have any TODO attached to it.